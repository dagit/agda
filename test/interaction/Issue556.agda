module Issue556 (A : Set) (x : A) where

y : A
y = x
