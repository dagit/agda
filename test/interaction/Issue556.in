top_command (cmd_load currentFile [])

-- Should be "A".
top_command (cmd_infer_toplevel Normalised "x")

-- Should be "A".
top_command (cmd_infer_toplevel Normalised "y")

-- Should be "x".
top_command (cmd_compute_toplevel False "x")

-- Should be "x".
top_command (cmd_compute_toplevel False "y")
