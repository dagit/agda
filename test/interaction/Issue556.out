Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue556 (Issue556.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue556.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> Agda2> Agda2> (agda2-status-action "Checked")
(agda2-info-action "*Inferred Type*" "A" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> Agda2> Agda2> (agda2-status-action "Checked")
(agda2-info-action "*Inferred Type*" "A" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> Agda2> Agda2> (agda2-status-action "Checked")
(agda2-info-action "*Normal Form*" "x" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> Agda2> Agda2> (agda2-status-action "Checked")
(agda2-info-action "*Normal Form*" "x" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 